lib/readline.js

```diff
@@ -429,7 +428,9 @@ Interface.prototype._tabComplete = function() {
           }
         }
         handleGroup(self, group, width, maxColumns);
+      }
 
+      {
```

Comment:
@yorkie Sorry, I’m not entirely sure what you’re suggesting – would you like to see the block removed? `let` vs `var` shouldn’t matter (but I would change `var` -> `let` if the block is removed, yes)